examples/SelfManagedProvider/SelfManagedProvider.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Absio.Sdk.Container;
+using Absio.Sdk.Providers;
+
+namespace SelfManagedProvider
+{
+    public class SelfManagedProvider : BaseProvider
+    {
+        public ServerCacheOfsProvider ServerCacheOfsProvider { get; }
+
+        public SelfManagedProvider()
+        {
+            ServerCacheOfsProvider = new ServerCacheOfsProvider();
+        }
+
+        /// <summary>
+        ///     Initialize the Container Provider.
+        /// </summary>
+        public void Initialize(string serverUrl, string apiKey, string applicationName, String ofsRootDirectory = null,
+            bool partitionDataByUser = false)
+        {
+            ServerCacheOfsProvider.Initialize(serverUrl, apiKey, applicationName, ofsRootDirectory,
+                partitionDataByUser);
+        }
+
+        /// <summary>
+        ///     Save the container data so that it is not stored on the server / ofs
+        /// </summary>
+        public async Task<SecuredContainer> CreateAsync(byte[] content, object customHeaderObject = null,
+            List<ContainerAccessLevel> accessLevels = null, string type = null)
+        {
+            var container = ServerCacheOfsProvider.ServerProvider.BuildNewContainer(content, customHeaderObject, accessLevels, type);
+            var securedContainer = await ServerCacheOfsProvider.ContainerEncryptionService.EncryptAsync(container);
+
+            var encryptedData = securedContainer.Bytes();
+            var newContainer = await SecuredContainer.CreateAsync(new byte[0], securedContainer.Metadata);
```

Comment:
You can now pass in null for the content of the secured container.  Please change to that.  Then there will be nothing created on the server or OFS regarding content files.